src/vs/workbench/contrib/tasks/common/jsonSchema_v2.ts

```diff
@@ -186,6 +186,10 @@ const group: IJSONSchema = {
 					type: 'boolean',
 					default: false,
 					description: nls.localize('JsonSchema.tasks.group.isDefault', 'Defines if this task is the default task in the group.')
+				},
+				glob: {
```

Comment:
Instead of having a separate `glob` property, let's make `isDefault` be `true`, `false`, or "glob". This lets us avoid groups like:
```json
"kind": "build",
"isDefault": "false",
"glob": "*.ts"
```
Since the goal of allowing a glob is to change the default, having `default === false` doesn't make sense when there is also a glob.

Reply:
Thanks @alexr00 is this something you was waiting on me to change or you were happy to make those changes?

Reply:
I was waiting on you to make the change :). I might get to it eventually, but my focus is usually on whatever our planned work for the milestone is first.